File: market-mapper/src/main/java/by/market/mapper/ProductCharacteristicMapper.java
package by.market.mapper;

import by.market.domain.characteristics.Characteristic;
import by.market.records.characteristics.ProductCharacteristicRecord;
import org.mapstruct.Mapper;

@Mapper(config = MapperConfiguration.class, uses = DataTypeMapper.class)
public interface ProductCharacteristicMapper extends MapstructMapper<ProductCharacteristicRecord, Characteristic> {
}

File: market-mapper/src/main/java/by/market/mapper/characteristics/ProductCharacteristicMapper.kt
package by.market.mapper.domain_dto_mapper.characteristics

import by.market.domain.characteristics.ProductCharacteristic
import by.market.dto.characteristics.ProductCharacteristicDTO
import by.market.mapper.IMapstructMapper
import by.market.mapper.MapperConfig
import org.mapstruct.Mapper

@Mapper(config = MapperConfig::class)
interface ProductCharacteristicMapper : IMapstructMapper<ProductCharacteristicDTO, ProductCharacteristic>